arangod/IResearch/IResearchFilterFactory.cpp

```diff
@@ -23,6 +23,8 @@
 ////////////////////////////////////////////////////////////////////////////////
 #include "Basics/DownCast.h"
 
+#include "frozen/map.h"
```

Comment:
```suggestion
#include <frozen/map.h>
```
I thought we use `<>` for third party

Reply:
Done